Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,258 @@
+﻿using System;
+using Console = System.Console;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+
+        public static void Demo()
+        {
+            bool moreEntries = true;
+
+            while (moreEntries)
+            {
+               // UserInfo();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4871920)